.vscode/settings.json

```diff
@@ -35,5 +35,5 @@
     "azure-pipelines.customSchemaFile": ".vscode/dnceng-schema.json",
     "dotnet.defaultSolution": "Roslyn.sln",
     "dotnet.completion.showCompletionItemsFromUnimportedNamespaces": true,
-    "dotnet.testWindow.disableAutoDiscovery": true
+    "dotnet.testWindow.disableAutoDiscovery": false
```

Comment:
>false [](http://example.com/codeflow?start=46&length=5)

Was this change intentional?

Reply:
Yes, though I can revert if you want to keep it separate.

Reply:
> Yes, though I can revert if you want to keep it separate.

I'm not sure what the change affects, but it looks like it's unrelated to overload resolution, so consider reverting.